static/js/tools/statsvar_menu.tsx

```diff
@@ -53,7 +54,10 @@ class Node extends Component<NodePropType, NodeStateType> {
     let child: JSX.Element[];
     let childCnt = 0;
     let svValid = false;
-    if (this.props.t === "v" && this.props.svValid.has(this.props.sv)) {
+    if (
```

Comment:
worth to make this a function and add some comments for the logic

Reply:
Added two functions to check whether the node can be expanded or checked.